cmd/object-common.go

```diff
@@ -225,7 +225,7 @@ func cleanupDir(storage StorageAPI, volume, dirPath string) error {
 		if err == errFileNotFound {
 			return nil
 		} else if err != nil { // For any other errors fail.
-			return traceError(err)
```

Comment:
> what was the reason you moved it down? to avoid stack info for a recursive call?

I just wanted to simplify. In this case, I guess I should make something like `return traceError(storage.DeleteFile(volume, entryPath))` in L219 (little above)